web-registry/src/components/organisms/BridgedEcocreditsTable/BridgedEcocreditsTable.tsx

```diff
@@ -0,0 +1,199 @@
+import { useState } from 'react';
+import { Box } from '@mui/material';
+import { quantityFormatNumberOptions } from 'config/decimals';
+import { random } from 'lodash';
+import { ELLIPSIS_COLUMN_WIDTH, tableStyles } from 'styles/table';
+
+import { BlockContent } from 'web-components/lib/components/block-content';
+import EmptyCartIcon from 'web-components/lib/components/icons/EmptyCartIcon';
+import ErrorIcon from 'web-components/lib/components/icons/ErrorIcon';
+import PendingIcon from 'web-components/lib/components/icons/PendingIcon';
+import SuccessIcon from 'web-components/lib/components/icons/SuccessIcon';
+import InfoLabel from 'web-components/lib/components/info-label';
+import {
+  ActionsTable,
+  DEFAULT_ROWS_PER_PAGE,
+  TablePaginationParams,
+} from 'web-components/lib/components/table/ActionsTable';
+import InfoTooltipWithIcon from 'web-components/lib/components/tooltip/InfoTooltipWithIcon';
+import { formatDate, formatNumber } from 'web-components/lib/utils/format';
+
+import { useLedger } from 'ledger';
+
+import {
+  AccountLink,
+  BreakText,
+  BreakTextEnd,
+  GreyText,
+  Link,
+} from 'components/atoms';
+import WithLoader from 'components/atoms/WithLoader';
+import { NoCredits } from 'components/molecules';
+import { useEcocredits } from 'hooks';
+
+import {
+  AMOUNT_BRIDGED_TOOLTIP,
+  BRIDGED_STATUS,
+  CREDIT_BATCH_TOOLTIP,
+  NO_BRIDGED_CREDITS,
+} from './BridgedEcocreditsTable.constants';
+import { iconStyle, withLoaderStyles } from './BridgedEcocreditsTable.styles';
+
+// TODO - Right now the data has been mocked using the ecocredit query.
+//      - Delete before merge.
+
+export const BridgedEcocreditsTable = (): JSX.Element => {
+  const { wallet } = useLedger();
+
+  const [paginationParams, setPaginationParams] =
+    useState<TablePaginationParams>({
+      page: 0,
+      rowsPerPage: DEFAULT_ROWS_PER_PAGE,
+      offset: 0,
+    });
+
+  const { credits, isLoadingCredits } = useEcocredits({
+    address: wallet?.address,
+    paginationParams,
+  });
+
+  const getStatusLabel = (status: string): JSX.Element | undefined => {
```

Comment:
This probably should be a component instead of a function to allow React to keep it between renders. We can also extract it to avoid it being redefined on each render.